firecares/settings/base.py

```diff
@@ -190,6 +190,7 @@
     'static_sitemaps',
     'import_export',
     'tastypie',
+    'mapwidgets'
 )
 
 OSGEO_IMPORTER = 'firecares.importers.GeoDjangoImport'
```

Comment:
`GOOGLE_API_KEY` configuration appears to be missing from the settings.  The `GOOGLE_API_KEY` setting should pull from the environment variable of the same name.